src/main/resources/META-INF/rust-core.xml

```diff
@@ -1093,14 +1095,18 @@
 
         <!-- Run Configurations -->
 
+        <programRunner implementation="org.rust.cargo.runconfig.customBuild.CustomBuildRunner"/>
```

Comment:
I think we should find a solution that uses `CompositeCargoRunConfigurationProducer`. But I'm not insisting to do it in this PR.